src/test/java/catalinaTema5/DemoqaElements/WebTables.java

```diff
@@ -0,0 +1,46 @@
+package catalinaTema5.DemoqaElements;
+
+import catalinaTema5.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class WebTables {
+    public static void main(String[] args) {
+        goToWebTablesPage();
+
+    }
+    public static void goToWebTablesPage() {
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/webtables");
+        WebElement searchButton = driver.findElement(By.id("basic-addon2"));
```

Comment:
This method throws an exception. Please double check how you identify the elements and make it pass. Thanks!